src/components/ConnectWallet/TorSetup.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="px-3 pb-2">
+    <p class="font-weight-bold">How to install Tor on your system</p>
+    <b-form-select v-model="selectedOS" class="mb-3">
+      <b-form-select-option :value="null" disabled
+        >Select your OS</b-form-select-option
+      >
+      <b-form-select-option value="macOS">macOS</b-form-select-option>
+      <b-form-select-option value="Linux">Linux</b-form-select-option>
+      <b-form-select-option value="Windows">Windows</b-form-select-option>
+    </b-form-select>
+
+    <ol class="pl-3" v-if="selectedOS == 'macOS'">
+      <li class="mb-3">
+        Open Terminal and install
+        <a href="https://brew.sh/" target="_blank">Homebrew</a>:
+        <input-copy
+          class="mt-2"
+          value='/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
+          size="sm"
+        ></input-copy>
+      </li>
+
+      <li class="mb-3">
+        Install Tor:
+        <input-copy
+          class="mt-2"
+          value="brew install tor"
+          size="sm"
+        ></input-copy>
+      </li>
+    </ol>
+
```

Comment:
Removed verification step (h/t @lukechilds) as we can't expect users to understand the output of verification.